### PR TITLE
Added functions, structs & constants to user32 & gdi32

### PR DESCRIPTION
**Pull request by recombinant**
Updated PR now reflects information from Windows SDK 7.1A

### CODE REVIEW

**Comment by retep998** [comment]
Could you rebase this PR against master?

**Comment by retep998** [comment]
`git pull --rebase origin master` where origin is your remote for this repo.

**Comment by recombinant** [comment]
done the rebase

**Comment by retep998** [comment]
Your PR is still full of merges commits.

**Comment by recombinant** [comment]
*$ git rebase -i HEAD~20*
(pick squash squash)
*$ git push origin master --force*
Now down to 1 commit.

File: src/wingdi.rs [hunk 1]
```diff
@@ -415,3 +415,173 @@ pub struct BLENDFUNCTION {
...
+pub const TA_LEFT: ::UINT = 0;
+pub const TA_RIGHT: ::UINT = 2;
+pub const TA_CENTER: ::UINT = 6;
+
+pub const TA_TOP: ::UINT = 0;
+pub const TA_BOTTOM: ::UINT = 8;
+pub const TA_BASELINE: ::UINT = 24;
+
+pub const TA_RTLREADING: ::UINT = 256;
+pub const TA_MASK: ::UINT = (TA_BASELINE+TA_CENTER+TA_UPDATECP+TA_RTLREADING);
```
Comment (retep998) [inline src/wingdi.rs]: Unnecessary parentheses and missing spaces around binary operators.

File: src/wingdi.rs [hunk 2]
```diff
@@ -415,3 +415,173 @@ pub struct BLENDFUNCTION {
...
+pub const OUT_PS_ONLY_PRECIS: ::DWORD = 10;
+
+pub const CLIP_DEFAULT_PRECIS: ::DWORD = 0;
+pub const CLIP_CHARACTER_PRECIS: ::DWORD = 1;
+pub const CLIP_STROKE_PRECIS: ::DWORD = 2;
+pub const CLIP_MASK: ::DWORD = 0xf;
+pub const CLIP_LH_ANGLES: ::DWORD = (1<<4);
+pub const CLIP_TT_ALWAYS: ::DWORD = (2<<4);
+pub const CLIP_DFA_DISABLE: ::DWORD = (4<<4);
+pub const CLIP_EMBEDDED: ::DWORD = (8<<4);
```
Comment (retep998) [inline src/wingdi.rs]: More unnecessary parentheses and missing spaces.

**Pull request closed without merging**